src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+shape of the profile is expected at the best fit
+values (\todo{Reference to modeling / fitting section?}).
+
+The result of the flux point estimation are either stored in a
+\code{FluxMaps} or \code{FluxPoints} object. Both objects
+are based on an internal representation of the flux which is
+independent of the SED type. The flux is represented by a
+the reference spectral model and an array of
+normalisation values given in energy, time and spatial bins,
+which factorise the deviation of the flux in the given
+bin form the reference spectral model. This allows
+user to conveniently transform between different
+SED types. Table~\ref{tab:sed_types} shows an
+overview and definitions of the supported SED types.
+The actual flux values for each SED type are obtained
+by mutiplication of the norm with the  reference flux.
+
+\begin{table*}
+    \begin{center}
+        \begin{tabular}{lll}
+         \hline
+         Type & Description & Unit Equivalency \\
+         \hline
+         dnde & Differential flux at a given energy & $\mathrm{TeV^{-1}~cm^{2}~s^{-1}}$ \\
+         e2dnde & Differential flux at a given energy  & $\mathrm{TeV~cm^{2}~s^{-1}}$ \\
+         flux & Integrated flux in agiven energy range & $\mathrm{cm^{2}~s^{-1}}$ \\
+         eflux & Integrated energy flux in agiven energy range & $\mathrm{erg~cm^{2}~s^{-1}}$\\
+         \hline
+        \end{tabular}
+    \end{center}
+    \label{tab:sed_types}
+    \caption{Definition of the different SED types supported in \gammapy.}
+\end{table*}
+
+
+Both result objects support the possibility to serialise
+the data into multiple formats. This includes the
+\gadf SED (reference?) format, FITS based ND sky-maps
+and formats compatible with Astropy's \code{Table} and
+\code{BinnedTimeSeries} data structures. This allows
+users to directly further analyse the results, e.g.
+standard algorithms for time analysis such as
+computing Lomb-Scargle periodograms or Bayesian
+blocks for time series. So far \gammapy does not
+support "unfolding" of \gammaray spectra.
+Methods for this will be implemented in future version of \gammapy.
+
+Code example ~\ref{fig*:minted:gp_estimators} shows how to use
+the \code{TSMapEstimator} objects with a given input \code{MapDataset}.
+In addition to the model it allows to specify the energy
+bins of the resulting flux and TS maps.
+
+
+\begin{figure}
+	\import{code-examples/generated/}{gp_estimators}
+	\caption{Using the \code{TSMapEstimator} object from \code{gammapy.estimators} to compute a
+        a flux, flux upper limits and TS map. The additional parameters \code{n\_sigma}
+        and \code{n\_sigma\_ul} define the confidence levels (in multiples of the normal distribution width)
+        of the flux error and and flux upper limit maps respectively.
+    }
+    \label{fig*:minted:gp_estimators}
+\end{figure}
+
+\subsection{gammapy.analysis}
+\label{ssec:gammapy-analysis}
+The \code{gammapy.analysis} sub-module provides a  high-level interface for the most
```

Comment:
not all the most common, is not it?
If not, I propose "for common use cases" ... being more quantitative